src/Components/ItemPanel.jsx

```diff
@@ -1,29 +1,43 @@
 import React, {useState} from 'react'
 import {makeStyles} from '@mui/styles'
+import IconButton from '@mui/material/IconButton'
 import ItemProperties from './ItemProperties'
 import ItemPropertiesDrawer from './ItemPropertiesDrawer'
-import MenuButton from '../Components/MenuButton'
+import Hamburger from '../assets/Icons/Menu.svg'
+import Close from '../assets/Icons/Close.svg'
 
 
 /**
- * Container for ItemProperties.  ItemProperties is wrapped in an
+ * Container for ItemProperties. ItemProperties is wrapped in an
  * ItemPropertiesDrawer to toggle hiding.
  * @param {Object} model IFC model
  * @param {Object} element The currently selected IFC element
  * @param {Number} topOffset Screen offset position
  * @return {Object} The ItemPanelButton react component
  */
-export default function ItemPanelButton({model, element, topOffset}) {
+export default function ItemPanelButton({model, element, topOffset, toggle, open}) {
```

Comment:
add @param docs for new params, and I think better names.. it's not clear what's being toggled or opened.. this component or something from the caller?